EventsExpress/Properties/launchSettings.json

```diff
@@ -17,7 +17,7 @@
       }
     },
     "EventsExpress": {
-      "commandName": "Project",
+      "commandName": "IISExpress",
```

Comment:
Consider not changing this file too much, or adding your own profile for iisexpress.